tests/test_core.py

```diff
@@ -27,7 +27,7 @@
 from common import skip_if, needs_dylink, no_windows, no_mac, is_slow_test, parameterized
 from common import env_modify, with_env_modify, disabled, node_pthreads, also_with_wasm_bigint
 from common import read_file, read_binary, require_v8, require_node
-from common import NON_ZERO, WEBIDL_BINDER, EMBUILDER
+from common import EMTEST_REBASELINE, NON_ZERO, WEBIDL_BINDER, EMBUILDER
```

Comment:
I think you need to reference EMTEST_REBASELINE vis its full name (`common.EMTEST_REBASELINE`).. otherwise the value you import can/will be stale.